libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -535,33 +536,56 @@ type UserClientAPI =
                :> "prekeys"
                :> Get '[JSON] [PrekeyId]
            )
-    :<|> NonceAPI
-
-type NonceAPI =
-  -- be aware that the order matters, if get was first, then head requests would be routed to the get handler
-  NewNonce "head-nonce" 'HEAD 200
+    -- be aware that the order of the head-nonce and get-nonce matters, if get was first, then head requests would be routed to the get handler
+    :<|> NewNonce "head-nonce" 'HEAD 200
     :<|> NewNonce "get-nonce" 'GET 204
+    :<|> Named
+           "create-access-token"
+           ( Summary "Create a JWT DPoP access token"
+               :> Description
+                    ( "Create an JWT DPoP access token for the client CSR, given a JWT DPoP proof, specified in the `DPoP` header. \
+                      \The access token will be returned as JWT DPoP token in the `DPoP` header."
+                    )
+               :> QualifiedCaptureUserId "uid"
+               :> "clients"
+               :> CaptureClientId "client"
+               :> "access-token"
```

Comment:
```suggestion
               :> Capture "cid" ClientId  -- (unless there is a dedicated combinator for that like for uid above?)
               :> "access-token"
```

Reply:
I don't understand, `CaptureClientId` is defined like this:

```haskell
type CaptureClientId name = Capture' '[Description "ClientId"] name ClientId
```

Reply:
doesn't make sense to me either.  either my suggestions was made on an outdated version of the code, or i was not paying much attention to what i wrote?  let's ignore me!